src/coreclr/System.Private.CoreLib/src/System/RuntimeHandles.cs

```diff
@@ -933,11 +947,11 @@ internal static string GetName(IRuntimeMethodInfo method)
         }
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        private static extern void* _GetUtf8Name(RuntimeMethodHandleInternal method);
+        private static extern void* GetUtf8NameInternal(RuntimeMethodHandleInternal method);
 
         internal static MdUtf8String GetUtf8Name(RuntimeMethodHandleInternal method)
         {
-            return new MdUtf8String(_GetUtf8Name(method));
+            return new MdUtf8String(GetUtf8NameInternal(method));
```

Comment:
```suggestion
            byte* name = GetUtf8NameInternal(method);
            if (name is null)
            {
                throw new BadImageFormatException();
            }
            return new MdUtf8String(name);
```
Uniform error handling